code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -23,10 +23,10 @@
 	name = "energy axe"
 	desc = "An energised battle axe."
 	icon_state = "axe0"
-	force = 40
-	force_on = 150
-	throwforce = 25
-	throwforce_on = 30
+	force = 52
+	force_on = 195

```

Comment:
nah it was 2 hit

this is 5 damage from instant death, and is effectively an instant kill